source/opt/ir_context.h

```diff
@@ -518,6 +518,14 @@ class IRContext {
         std::string message = "ID overflow. Try running compact-ids.";
         consumer()(SPV_MSG_ERROR, "", {0, 0, 0}, message.c_str());
       }
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+      // If TakeNextId returns 0, it is very likely that execution will
+      // subsequently fail. Such failures are false alarms from a fuzzing point
+      // of view: they are due to the fact that too many ids were used, rather
+      // than being due to an actual bug. Thus, during a fuzzing build, it is
+      // preferable to bail out when ID overflow occurs.
+      exit(0);
```

Comment:
Did we confirm that we cannot exit with an error code of some type?

Reply:
I'll double check that.

Reply:
I got confirmation that a non-zero exit code will indeed be regarded as a crash, so we need to stick with exit(0) here. I have elaborated on this in the comment.

Reply:
Okay.